### PR TITLE
[chartmuseum] move from incubator to stable

### PR DESCRIPTION
**Pull request by jdolitsky**
Also, adding new Kubernetes org member @goruha to OWNERS

@sameersbn @prydonius this is a rehash of #3314 back in January. We have since removed the dependency on common chart.

### CODE REVIEW

File: stable/chartmuseum/README.md
Comment (sameersbn) [inline stable/chartmuseum/README.md]: should be updated to `stable/chartmuseum` in the README

File: stable/chartmuseum/README.md
Comment (sameersbn) [inline stable/chartmuseum/README.md]: Please remove the extra new line characters

File: stable/chartmuseum/templates/NOTES.txt
Comment (sameersbn) [inline stable/chartmuseum/templates/NOTES.txt]: Can we have some additional notes on how the user can access the deployment or reach the API?

File: stable/chartmuseum/values.yaml
Comment (sameersbn) [inline stable/chartmuseum/values.yaml]: Unless absolutely required we do not recommend enforcing resource requests/limits on charts. Should ideally be:

```
resource: {}
```

File: stable/chartmuseum/values.yaml
Comment (sameersbn) [inline stable/chartmuseum/values.yaml]: we should use consistent naming. Please use camelCase for the keys here as well

**Review by sameersbn — CHANGES_REQUESTED** [review]

File: stable/chartmuseum/values.yaml
Comment (unguiculus) [reply stable/chartmuseum/values.yaml]: Keys should start with a lower case character. I guess that's what you mean @sameersbn?

File: stable/chartmuseum/templates/deployment.yaml
Comment (unguiculus) [inline stable/chartmuseum/templates/deployment.yaml]: Does it make sense to have the container port configurable? Just wondering. Also, you could give it a name and reference it by its name from service and probes.

**Review by unguiculus — CHANGES_REQUESTED** [review]

**Comment by unguiculus** [comment]
There seems to be an issue with a recent change in CI scripts. Maintainers seem to always be required but we want them to be removed from deprecated charts.

See #4161.

File: stable/chartmuseum/values.yaml
Comment (sameersbn) [reply stable/chartmuseum/values.yaml]: yes. thats correct. the entire `persistence:` block uses TitleCase breaking consistency

File: stable/chartmuseum/Chart.yaml
Comment (prydonius) [inline stable/chartmuseum/Chart.yaml]: should this be a major version bump (1.0.0?). I think a minor version bump is okay for `0.` releases, but perhaps the major version would be more indicative of some of these proposed value changes.

File: stable/chartmuseum/templates/secret.yaml
Comment (scottrigby) [inline stable/chartmuseum/templates/secret.yaml]: Needs newline at EOF

File: stable/chartmuseum/Chart.yaml
Comment (scottrigby) [reply stable/chartmuseum/Chart.yaml]: I was wondering something similar. If we are changing helm values YAML keys, then there's backwards incompatibility at those points, so that requires a semver MAJOR bump.

**Comment by jdolitsky** [comment]
@prydonius @sameersbn @unguiculus @scottrigby we have addressed your comments.

@goruha made several changes in the incubator chart involving values etc., and that version has been set to 1.0.0. Now it will be deprecated, and set to 1.0.1, and the new stable version will be 1.1.0

**Comment by jdolitsky** [comment]
@unguiculus -
> There seems to be an issue with a recent change in CI scripts. Maintainers seem to always be required but we want them to be removed from deprecated charts.

Are we blocked on merging this PR until that is fixed? Should I change incubator to `maintainers: []`?

**Review by unguiculus — APPROVED** [review]

**Pull request merged**
